### PR TITLE
ERC721 subgraph

### PR DESCRIPTION
**Pull request by corerouter**

### CODE REVIEW

**Comment by 0xbe1** [comment]
LGTM

**Comment by mrbrianhobo** [comment]
also looks good to me!

**Comment by this-username-is-taken** [comment]
Looks great! Just one thing: can you rename `TransferEvent` to `Transfer`?

File: subgraphs/erc721/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+  " timestamp of the token's mint "
+  mintTime: BigInt!
+}
+
+#############################################
+####### ERC721 Smart Contract Metadata ######
+#############################################
+
+type Contract @entity {
```
Comment (this-username-is-taken) [inline subgraphs/erc721/schema.graphql:33]: Can you rename this to `Collection`?

Comment (corerouter) [reply subgraphs/erc721/schema.graphql]: Yes. Modified at https://github.com/messari/subgraphs/pull/689/commits/94a0e318f6371aaff194d76f3e621577de2d5f69

**Comment by corerouter** [comment]
> Looks great! Just one thing: can you rename `TransferEvent` to `Transfer`?
Yes. Modified at 

https://github.com/messari/subgraphs/pull/689/commits/94a0e318f6371aaff194d76f3e621577de2d5f69

**Pull request merged**